### PR TITLE
add thrush combinator as applyTo

### LINKED ISSUE #1846: add the T combinator : R.thrush ?
Hey guys, I think add a thrush function in Ramda can be cool, is not it ?
`
const thrush = x => f => f(x);
`

an example here with lenses: http://goo.gl/RgLubh

### PR DESCRIPTION
**Pull request by kedashoe**
Closes #1846, #1847

### CODE REVIEW

File: source/index.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export { default as aperture } from './aperture';
 export { default as append } from './append';
 export { default as apply } from './apply';
 export { default as applySpec } from './applySpec';
+export { default as applyTo } from './applyTo';
```
Comment (Bradcomp) [inline source/index.js]: Is this something we will need to add into our PRs in the future? Can we get a build script to do this?

**Comment by buzzdecafe** [comment]
:cow2:

File: source/index.js (hunk 1 above)
Comment (kedashoe) [reply source/index.js]: I was in favor of doing this automatically via a script when we first added an index.js file but we decided against it.. I think it is fine to do it manually as I believe in all cases (adding, moving, deleting functions) either tests or building will fail if index.js is not updated.

Comment (Bradcomp) [reply source/index.js]: It may be worth it at this point to update the instructions in CONTRIBUTING.md

**Pull request merged**